src/Resources/contao/classes/BackendUser.php

```diff
@@ -154,39 +158,6 @@ public function __get($strKey)
 		return parent::__get($strKey);
 	}
 
-
-	/**
-	 * Redirect to the login screen if authentication fails
-	 *
-	 * @return boolean True if the user could be authenticated
-	 */
-	public function authenticate()
```

Comment:
The method is back in there - I've got it wrong in the first view. 
If we now use `BackendUser::authenticate()` we are compatible to all other backend entry points (see https://github.com/contao/core-bundle/pull/559/commits/58e6b65c9569fd70532332118dbdb7baac834bcf).